code/game/objects/structures/watercloset.dm

```diff
@@ -235,6 +235,9 @@
 	icon_state = "rubberducky"
 	item_state = "rubberducky"
 
+/obj/item/bikehorn/rubberducky/plasticducky
+	name = "plastic ducky"
+	desc = "It's a cheap plastic knockoff of a loveable bathtime toy."
```

Comment:
set custom_materials on this to make sure the spawned ones have plastic too

Reply:
Thanks

Reply:
Why'd you remove it? 364fbb9